bin/pycbc_optimize_snr

```diff
@@ -280,6 +314,9 @@ parser.add_argument('--pso-c2', type=float, default=2.0,
 parser.add_argument('--pso-w', type=float, default=0.01,
                     help='Only relevant for --optimizer pso: '
                     'The hyperparameter w: the inertia parameter.')
+parser.add_argument('--include-candidate-in-optimizer', action='store_true',
+                    help='Include parameters of the candidate event in the '
+                    'initialised array for the optimizer')
```

Comment:
I would like to keep this as an option, basically because there could be some ever-so-slight bias introduced here, so we should monitor the results and check if they are sensible

Reply:
Sure, I guess in the production run this option should be enabled but it is useful to keep this as an option for any testing in the future with improving this code.